packages/widgets/src/components/NotificationCard/types.ts

```diff
@@ -1,11 +1,8 @@
-import { IMapIcons, MainColors } from '@ssa-ui-kit/core';
-
 export interface NotificationCardProps {
   title: string;
   text: string;
   isRead: boolean;
-  badgeColor: keyof MainColors;
-  iconName: keyof IMapIcons;
+  type: string;
```

Comment:
I suggest making the `type` more strict: support only those values we support in the component: Warning, Informational, Error.

Reply:
Done